### DISCUSSION TITLE
Interesting neural network compression technology

### DISCUSSION THREAD

**Discussion by igor-yusupov** [robertknight/rten#461] [Show and tell]
Hi! I found an interesting service for running language models with 8 billion parameters on user devices. Maybe it will be useful for you and you will find some tricks for your engine: [link](https://galqiwi.github.io/aqlm-rs/about.html) and [code](https://github.com/galqiwi/demo-aqlm-rs/tree/demo)

**Comment by robertknight** [comment]
Looks interesting. Thanks for the info!
